### PR TITLE
Add metadata transfer example. 

### LINKED ISSUE #145: Add metadata transfer example.

### PR DESCRIPTION
**Pull request by misselvexu**
## PR Type
Add feature example & documentation content changes .

## Describe what this PR does for and how you did.

- Add metedata transfer example modules .

## Does this PR be associated with issue? If so, please adding the issue link below.
Fixed #145 .

### Checklist
- [x] Code compiles correctly
- [ ] Pull Request has submit to 2020.0 and Greenwich
- [ ] Create at least one junit test if possible
- [x] All tests passing
- [x] Extend documentation if necessary
- [ ] Add myself / the copyright holder to the AUTHORS file

### CODE REVIEW

File: spring-cloud-tencent-examples/polaris-metadata-transfer-example/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Spring Cloud Polaris Metadata Transfer example
+
+## Example Introduction
+
+This example shows how to use ```spring-cloud-starter-tencent-polaris-metadata-transfer`` in Spring Cloud project for its features.
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/polaris-metadata-transfer-example/README.md:5]: 修改成正确的模块名

File: spring-cloud-tencent-examples/polaris-metadata-transfer-example/README-zh.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# Spring Cloud Polaris Metadata Transfer example
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/polaris-metadata-transfer-example/README-zh.md:1]: Metadata与Polaris功能无关，去掉标题的Polaris

File: spring-cloud-tencent-examples/polaris-metadata-transfer-example/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+# Spring Cloud Polaris Metadata Transfer example
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/polaris-metadata-transfer-example/README.md:1]: Metadata与Polaris功能无关，去掉标题的Polaris

File: spring-cloud-tencent-examples/polaris-metadata-transfer-example/metadata-callee-service/pom.xml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+		<artifactId>polaris-metadata-transfer-example</artifactId>
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/polaris-metadata-transfer-example/metadata-callee-service/pom.xml:6]: Metadata与Polaris功能无关，去掉Polaris

File: spring-cloud-tencent-examples/polaris-metadata-transfer-example/metadata-callee-service/pom.xml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        <groupId>com.tencent.cloud</groupId>
+        <version>${revision}</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>metadata-callee-service</artifactId>
+    <name>Polaris Metadata Transfer Callee Service</name>
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/polaris-metadata-transfer-example/metadata-callee-service/pom.xml:14]: Metadata与Polaris功能无关，去掉Polaris

File: spring-cloud-tencent-examples/polaris-metadata-transfer-example/metadata-caller-service/pom.xml [hunk 6]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+		<artifactId>polaris-metadata-transfer-example</artifactId>
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/polaris-metadata-transfer-example/metadata-caller-service/pom.xml:6]: Metadata与Polaris功能无关，去掉Polaris

File: spring-cloud-tencent-examples/polaris-metadata-transfer-example/pom.xml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>spring-cloud-tencent-examples</artifactId>
+		<groupId>com.tencent.cloud</groupId>
+		<version>${revision}</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>polaris-metadata-transfer-example</artifactId>
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/polaris-metadata-transfer-example/pom.xml:13]: Metadata与Polaris功能无关，去掉Polaris

File: spring-cloud-tencent-examples/pom.xml [hunk 8]
```diff
@@ -22,6 +22,7 @@
         <module>polaris-gateway-example</module>
 		<module>polaris-config-example</module>
 		<module>polaris-router-example</module>
+		<module>polaris-metadata-transfer-example</module>
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-examples/pom.xml:25]: Metadata与Polaris功能无关，去掉Polaris

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]

**Review by SkyeBeFreeman — APPROVED** [review]
LGTM.

**Comment by SkyeBeFreeman** [comment]
@misselvexu please check the failing check.

**Comment by misselvexu** [comment]
> @misselvexu please check the failing check.

Resolved .

**Comment by SkyeBeFreeman** [comment]
@misselvexu Add this PR to [CHANGELOG.md](https://github.com/Tencent/spring-cloud-tencent/blob/main/CHANGELOG.md) and cherry-pick this PR to branch greenwich and 2020.0.

**Comment by SkyeBeFreeman** [comment]
@misselvexu 建议合并这次PR的多次commit为一次commit，然后强制提交以修改本次PR中的commit的历史。

**Comment by misselvexu** [comment]
> @misselvexu 建议合并这次PR的多次commit为一次commit，然后强制提交以修改本次PR中的commit的历史。

Fixed .

**Review by SkyeBeFreeman — APPROVED** [review]
LGTM.

**Comment by misselvexu** [comment]
> @misselvexu Add this PR to [CHANGELOG.md](https://github.com/Tencent/spring-cloud-tencent/blob/main/CHANGELOG.md) and cherry-pick this PR to branch greenwich and 2020.0.

After cherry-pick , Need to submit another PR for branch greenwich and 2020.0? Because some class API is changed.

**Comment by SkyeBeFreeman** [comment]
> > @misselvexu Add this PR to [CHANGELOG.md](https://github.com/Tencent/spring-cloud-tencent/blob/main/CHANGELOG.md) and cherry-pick this PR to branch greenwich and 2020.0.
> 
> After cherry-pick , Need to submit another PR for branch greenwich and 2020.0? Because some class API is changed.

It is needed to merge to branch greenwich and 2020.0. This PR has changes between greenwich, 2020.0 with hoxton?

**Pull request merged**